packages/utils/upgrade/resources/codemods/5.0.0/sqlite3-to-better-sqlite3.json.ts

```diff
@@ -0,0 +1,37 @@
+import path from 'node:path';
+
+import type { modules } from '../../../dist';
+
+/**
+ * This codemod runs on the package.json of the Strapi project and updates
+ * the sqlite dependency to better-sqlite3
+ */
+const transform: modules.runner.json.JSONTransform = (file, params) => {
+  const { cwd, json } = params;
+
+  const rootPackageJsonPath = path.join(cwd, 'package.json');
+
+  if (file.path !== rootPackageJsonPath) {
+    return file.json;
+  }
+
+  const j = json(file.json);
+
+  const targetProperties = ['sqlite3', 'vscode/sqlite3', 'sqlite-legacy'];
```

Comment:
- Shouldn't `'vscode/sqlite3'` be `'@vscode/sqlite3'`? (https://www.npmjs.com/package/@vscode/sqlite3)
- I can't find any release for `'sqlite-legacy'`. Where do this come from?